language/fr.json

```diff
@@ -0,0 +1,480 @@
+    "Forced Overwrite of Generating Height": "Écrasement forcé de la hauteur de génération",
+    "Forced Overwrite of Denoising Strength of \"Vary\"": "Écrasement forcé du débruitage Force de \"Variable\"",
+    "Set as negative number to disable. For developer debugging.": "Définir un nombre négatif pour désactiver. Pour le débogage des développeurs.",
+    "Forced Overwrite of Denoising Strength of \"Upscale\"": "Écrasement forcé du débruitage Force de \"Mise à l'échelle\"",
+    "Inpaint Engine": "Moteur d'Inpaint",
+    "v1": "v1",
+    "Version of Fooocus inpaint model": "Version du modèle Fooocus inpaint (peinture)",
+    "v2.5": "v2.5",
+	"Inpaint Denoising Strength": "Intensité du débruitage Inpaint (peinture)",
+	"Same as the denoising strength in A1111 inpaint. Only used in inpaint, not used in outpaint. (Outpaint always use 1.0)": "Identique à la force de débruitage dans l'inpaint A1111. Utilisé uniquement en inpaint, pas en outpaint. (Outpaint utilise toujours 1.0)",
+	"Inpaint Respective Field": "Champ respectif Inpaint (peinture)",
+	"The area to inpaint. Value 0 is same as (Only Masked) in A1111. Value 1 is same as (Whole Image) in A1111. Only used in inpaint, not used in outpaint. (Outpaint always use 1.0)": "La zone à peindre dedans. La valeur 0 est identique à la valeur (Masqué uniquement) de l'article A1111. La valeur 1 est identique à (Image entière) dans A1111. Utilisé uniquement pour la peinture, pas pour Peindre en dehors. (Peindre en dehors utilise toujours 1.0)",
+	"Mask Erode or Dilate": "Masque éroder ou dilater",
+	"Positive value will make white area in the mask larger, negative value will make white area smaller.(default is 0, always process before any mask invert)": "Une valeur positive agrandit la zone blanche du masque, une valeur négative la réduit. (La valeur par défaut est 0, le traitement est toujours effectué avant l'inversion du masque).",
+	"Enable Mask Upload": "Activer le chargement du masque",
+	"Invert Mask": "Inverser le masque",
+    "Control Debug": "Contrôle Débogage",
+	"See the results from preprocessors.": "Voir les résultats des préprocesseurs.",
+    "Debug Preprocessors": "Préprocesseurs de débogage",
+	"Do not preprocess images. (Inputs are already canny/depth/cropped-face/etc.)": "Ne pas prétraiter les images. (Les données d'entrée sont déjà de bonne qualité/profondeur/face recadré/etc.)",
+	"Skip Preprocessors": "Passer les préprocesseurs",
+    "Mixing Image Prompt and Vary/Upscale": "Mélange du Prompt à l'image et de Variable/Mise à l'échelle",
+    "Mixing Image Prompt and Inpaint": "Mélange entre le prompt d'image et peindre dedans",
+	"ImagePrompt": "Prompt d'image",
+	"FaceSwap": "Permute les faces",
```

Comment:
```suggestion
	"FaceSwap": "Permute les visages",
```

Reply:
Sorry for the wait. The modifications suggested I had in fact already made a few days ago. I just forgot to update on Github